src/Runners/PHPUnit/Options.php

```diff
@@ -1078,14 +1090,19 @@ public function logJunit(): ?string
         return $this->logJunit;
     }
 
+    public function teamcity(): bool
+    {
+        return $this->teamcity;
+    }
+
     public function logTeamcity(): ?string
     {
         return $this->logTeamcity;
     }
 
     public function hasLogTeamcity(): bool
     {
-        return $this->logTeamcity !== null;
+        return $this->teamcity || $this->logTeamcity !== null;
```

Comment:
Output and logging are two different things and shouldn't be coupled

Reply:
This point doesn't seem resolved to me: a user could use `--teamcity` but not `--log-teamcity` and viceversa, please split the two behaviors

Reply:
Hm, this seems to be already changed, see:
https://github.com/paratestphp/paratest/pull/597/files#diff-e7eb5918ab64b6a45aac1cb6586bd195c5a96fd600882a7be2478afe58edbe5cR1107

The resulting code looks like this:
```
...
public function hasLogTeamcity(): bool
{
    return $this->logTeamcity !== null;
}

public function needsTeamcity(): bool
{
    return $this->teamcity() || $this->hasLogTeamcity();
}
...
```